py/objmodule.c

```diff
@@ -230,6 +230,9 @@ STATIC const mp_rom_map_elem_t mp_builtin_module_table[] = {
     #if MICROPY_PY_BLUETOOTH
     { MP_ROM_QSTR(MP_QSTR_ubluetooth), MP_ROM_PTR(&mp_module_ubluetooth) },
     #endif
+    #if MICROPY_PY_PLATFORM
+    { MP_ROM_QSTR(MP_QSTR_platform), MP_ROM_PTR(&mp_module_platform) },
```

Comment:
This is an interesting example... should this be `MP_QSTR_uplatform` ? I don't know...

(Would anyone want to write a platform.py that provides additional functionality... This is a rare case where we provide exactly the whole feature set of a CPython module.)

Reply:
I think it should be `uplatform`, if only to support the case of overriding values.  But also I don't think we need to provide exactly the whole set of CPython functions, and so `platform.py` could add those missing ones.